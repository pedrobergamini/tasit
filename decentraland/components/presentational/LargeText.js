import React from "react";
import { StyleSheet, Text } from "react-native";
import {
  responsiveHeight,
  responsiveFontSize,
} from "react-native-responsive-dimensions";
import Colors from "@shared-constants/Colors";
import PropTypes from "prop-types";

export default function LargeText(props) {
  return <Text style={styles.text}>{props.children}</Text>;
}

LargeText.propTypes = {
  children: PropTypes.string.isRequired,
};

const styles = StyleSheet.create({
  text: {
    padding: responsiveHeight(2),
    fontSize: responsiveFontSize(4),
    textAlign: "center",
    fontWeight: "bold",
    color: Colors.textColor,
  },
});
