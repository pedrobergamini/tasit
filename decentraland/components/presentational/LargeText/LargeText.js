import React from "react";
import { StyleSheet, Text } from "react-native";
import {
  responsiveHeight,
  responsiveFontSize,
} from "react-native-responsive-dimensions";
import Colors from "@shared-constants/Colors";
import PropTypes from "prop-types";

export default function LargeText(props) {
  return <Text style={styles.text}>{props.children}</Text>;
}

LargeText.propTypes = {
  children: PropTypes.string.isRequired,
};

const styles = StyleSheet.create({
  text: {
    color: Colors.textColor,
    fontSize: responsiveFontSize(4),
    fontWeight: "bold",
    padding: responsiveHeight(2),
    textAlign: "center",
  },
});
