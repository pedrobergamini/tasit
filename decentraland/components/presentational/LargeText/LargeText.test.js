import React from "react";
import { shallow } from "enzyme";
import LargeText from "@presentational/LargeText";

describe("LargeText", () => {
  it("renders the component", async () => {
    expect(shallow(<LargeText>{""}</LargeText>)).toMatchSnapshot();
  });
});
