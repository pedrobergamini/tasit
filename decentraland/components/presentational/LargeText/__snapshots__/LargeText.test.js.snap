// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LargeText renders the component 1`] = `
<Component
  style={
    Object {
      "color": "#CCCCCC",
      "fontSize": 61.191865835619396,
      "fontWeight": "bold",
      "padding": 26.68,
      "textAlign": "center",
    }
  }
/>
`;
